vignettes/release-checklists.qmd

```diff
@@ -0,0 +1,110 @@
+---
+title: "Release Checklists"
+description: >
+  Reference for how to perform R package releases in the Hubverse
+vignette: >
+  %  \VignetteIndexEntry{Release Checklists}
+  %  \VignetteEngine{quarto::html}
+  %  \VignetteEncoding{UTF-8}
+knitr:
+  opts_chunk: 
+    collapse: true
+    comment: '#>'
+---
+
+## Release Checklist
+
+ - [ ] Create a new branch from `main` called `<author>/release/<X.Y.Z>` (e.g. `ls/release/0.1.6`)
+ - [ ] Update version number in DESCRIPTION and check that the `Remotes:` is
+       up-to-date
+ - [ ] Update NEWS for the changes in this version
```

Comment:
Should individual non-tagged releases carry NEWS updates with them? Do we have guidance on whether these NEWS updates will be present already or whether the releaser will need to go through and write all of this up?

Reply:
Yes and yes in [lines 58 and 59](https://github.com/hubverse-org/hubDevs/pull/11/files/1ee04ba466e3ef849c9c5a1878ec6be5c4ac8662#diff-274d4788e65713582311e17f54c220db7e5bbdbf75d2d43afa425232520e8852R58-R59).

Also, I would avoid using the term "non-tagged releases" because if they aren't tagged, they aren't released. I would just call them pull requests.